include/cloud_on.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+source $(dirname $0)/functions.sh
+
+cloud_init()
+{
+    # only ON is supported atm
+    CONFIG_DIR=${SECANT_CONFIG_DIR:-/etc/secant}
+    source ${CONFIG_DIR}/cloud.conf
+
+    export ONE_HOST ONE_XMLRPC
+}
+
+cloud_shutdown_vm()
+{
+    logging $TEMPLATE_IDENTIFIER "Shutting down VM $VM_ID." "DEBUG"
+    onevm shutdown --hard $VM_ID
+    if [ $? -ne 0 ]; then
+        logging $TEMPLATE_IDENTIFIER "Failed to shutdown VM $VM_ID." "ERROR"
+    fi
```

Comment:
Return a status code (the result of the one* command in most cases)

The note holds for all the functions here, actually